apis/apis.json

```diff
@@ -740,13 +740,13 @@
   },
 
   {
-    "id": "Google.Cloud.Secrets.V1Beta1",
+    "id": "Google.Cloud.SecretManager.V1Beta1",
     "generator": "micro",
     "protoPath": "google/cloud/secrets/v1beta1",
```

Comment:
And the proto path hasn't changed?

Reply:
Proto path hasn't changed. Will fix the readme in another PR later.

Reply:
Proto path hasn't changed. Will fix the readme in another PR later.